src/core_plugins/kibana/public/visualize/editor/editor.js

```diff
@@ -77,16 +76,29 @@ function VisEditor($scope, $route, timefilter, AppState, $location, kbnUrl, $tim
   let stateMonitor;
   const $appStatus = this.appStatus = {};
 
+  // Retrieve the resolved SavedVis instance.
   const savedVis = $route.current.locals.savedVis;
 
+  // Instance of vis.js.
   const vis = savedVis.vis;
+
+  // Clone the _vis instance.
   const editableVis = vis.createEditableVis();
+
+  // We intend to keep editableVis and vis in sync with one another, so calling `requesting` on
+  // vis should call it on both.
   vis.requesting = function () {
     const requesting = editableVis.requesting;
+    // Invoking requesting() calls onRequest on each agg's type param. When a vis is marked as being
+    // requested, the bounds of that vis are updated and new data is fetched using the new bounds.
     requesting.call(vis);
+
+    // The benefit of calling `editableVis.requesting()` isn't clear.
```

Comment:
You actually made this comment on the original PR: https://github.com/elastic/kibana/pull/7968#r74357613